### PR TITLE
[cms] Add domain invoices page for all users

### LINKED ISSUE #2014: [cms] Add domain invoices page for all users
Requires: https://github.com/decred/politeia/pull/1222

This new page will allow users to see all invoices from their equal domain.  It will appear pretty much exactly like the current admin invoices page (minus the invite/payout links).

The same request is currently being used: admin/invoices.  If not an admin it will return only invoices created by a user in the same domain.  These return invoice records have been stripped of any private details and should only show line items with description/proposal id and labor hours.

### PR DESCRIPTION
**Pull request by amassarwi**
This diff updates the invoices list `admin/invoices` to `/invoices` as it now 
available to normal cms contractors, the invoices page content should change 
according to logged in user & should display admins only content (e.g invite 
contractor, payouts, etc.)

Contractors shouldn't see sensitive info like hour rate & dcr paid of each other. 

### Solution description
Tweaked existing route/page, and added some user type specific logic

### Dependencies
Need  https://github.com/decred/politeia/pull/1222
Closes #2014 

### Screenshot
image

### CODE REVIEW

**Comment by alexlyp** [comment]
@amassarwi ok politeia #1222 has been merged.

Can you rebase this so we can do final testing?

**Comment by amassarwi** [comment]
Sure, will do in a bit

**Comment by alexlyp** [comment]
After a little more discussion in contractors I'd like to wait until https://github.com/decred/politeia/pull/1275 and https://github.com/decred/politeia/pull/1276 are merged to finalize this.

**Comment by alexlyp** [comment]
OK those PRs are merged so this PR's testing may proceed

**Comment by amassarwi** [comment]
Gonna test it tonight against latest BE master

File: src/pages/RoutesCMS.jsx [hunk 1]
```diff
@@ -70,6 +70,12 @@ const Routes = ({ location }) => {
             exact
             component={PageInvoicesNew}
           />
+          <AuthenticatedRoute
+            path="/invoices"
+            title="All invoices" // XXX: we could should show differnt title for admins maybe?
```
Comment (tiagoalvesdulce) [inline src/pages/RoutesCMS.jsx:75]: hmm, maybe we could show `Domain invoices` to contractors because they will be able to see only their domain invoices.

Comment (amassarwi) [reply src/pages/RoutesCMS.jsx:75]: good point, will do

**Pull request merged**